package.json

```diff
@@ -11,14 +11,14 @@
     "url": "git@github.com:elasticio/sftp-component.git"
   },
   "engines": {
-    "node": "14"
+    "node": "16"
   },
   "author": "elastic.io GmbH",
   "license": "Apache-2.0",
   "dependencies": {
-    "@elastic.io/component-commons-library": "2.0.1",
+    "@elastic.io/component-commons-library": "2.1.0-dev.6",
```

Comment:
on't forget to change the lib version after testing the component

Reply:
done